### PR TITLE
EX-SU-IfcArchElement/STEELRIB and Pset_ArchElementTypeSteelRib

### PR DESCRIPTION
**Pull request by larswik**

### CODE REVIEW

File: IFC4x3/Properties/b/BoltType_2VFVwrp3T3Sf2eWFjBM4gw/DocProperty.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="BoltType_2VFVwrp3T3Sf2eWFjBM4gw" Name="BoltType" UniqueId="9f3dfeb5-cc37-4372-90a8-80fb4b584aba" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/b/BoltType_2VFVwrp3T3Sf2eWFjBM4gw/DocProperty.xml]: Missing PrimaryDataType.

File: IFC4x3/Properties/f/FootOfArchs_2qRbpwn216hwkxu4hPpUVh/DocProperty.xml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="FootOfArchs_2qRbpwn216hwkxu4hPpUVh" Name="FootOfArchs" UniqueId="b46e5cfa-c420-46af-abbb-e04ad9cde7eb" PrimaryDataType="IfcLabel" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/f/FootOfArchs_2qRbpwn216hwkxu4hPpUVh/DocProperty.xml]:
```suggestion
<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="FootOfArchs_2qRbpwn216hwkxu4hPpUVh" Name="FootOfArches" UniqueId="b46e5cfa-c420-46af-abbb-e04ad9cde7eb" PrimaryDataType="IfcLabel" />
```

File: IFC4x3/Properties/l/LongitudinalConnections_1azFwlJa58zglMh9JjImfr/DocProperty.xml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="LongitudinalConnections_1azFwlJa58zglMh9JjImfr" Name="LongitudinalConnections" UniqueId="64f4feaf-4e41-48f6-abd6-ac94ed4b0a75" PrimaryDataType="IfcLabel" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/l/LongitudinalConnections_1azFwlJa58zglMh9JjImfr/DocProperty.xml:2]: No description or reusing?

File: IFC4x3/Properties/l/LongitudinalSpacing_2Fqc$pMNj5C8jRJPnj8nSC/DocProperty.xml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="LongitudinalSpacing_2Fqc$pMNj5C8jRJPnj8nSC" Name="LongitudinalSpacing" UniqueId="8fd26ff3-597b-4530-8b5b-4d9c6d23170c" PrimaryDataType="IfcPositiveLengthMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/l/LongitudinalSpacing_2Fqc$pMNj5C8jRJPnj8nSC/DocProperty.xml:2]: Are the still ribs not modelled i.e. can be extracted from geometry?

File: IFC4x3/Properties/t/TypeOfReticularArch_08Yo2Ip_fAMRuKPkd8Ied2/DocProperty.xml [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="TypeOfReticularArch_08Yo2Ip_fAMRuKPkd8Ied2" Name="TypeOfReticularArch" UniqueId="088b2092-cfea-4a59-be14-66e9c84a89c2" PrimaryDataType="IfcLabel" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/t/TypeOfReticularArch_08Yo2Ip_fAMRuKPkd8Ied2/DocProperty.xml]:
```suggestion
<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="TypeOfReticularArch_08Yo2Ip_fAMRuKPkd8Ied2" Name="ReticularArchType" UniqueId="088b2092-cfea-4a59-be14-66e9c84a89c2" PrimaryDataType="IfcLabel" />
```

File: IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedBldgElements/PropertySets/Pset_ArchElementTypeSteelRib/Documentation.md [hunk 6]
```diff
@@ -0,0 +1 @@
+Properties for arch elements of type STEELRIB.
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedBldgElements/PropertySets/Pset_ArchElementTypeSteelRib/Documentation.md]:
```suggestion
Properties for arch elements of type _STEELRIB_.
```

**Review by SergejMuhic — CHANGES_REQUESTED** [review]

File: IFC4x3/Properties/b/BoltType_2VFVwrp3T3Sf2eWFjBM4gw/DocProperty.xml (hunk 1 above)
Comment (larswik) [reply IFC4x3/Properties/b/BoltType_2VFVwrp3T3Sf2eWFjBM4gw/DocProperty.xml]: Yes! This one should reuse the BoltType from #827. Just added it here to remember.

File: IFC4x3/Properties/l/LongitudinalConnections_1azFwlJa58zglMh9JjImfr/DocProperty.xml (hunk 3 above)
Comment (larswik) [reply IFC4x3/Properties/l/LongitudinalConnections_1azFwlJa58zglMh9JjImfr/DocProperty.xml:2]: No reuse, but missing description. @JontyDunn.

File: IFC4x3/Properties/l/LongitudinalSpacing_2Fqc$pMNj5C8jRJPnj8nSC/DocProperty.xml (hunk 4 above)
Comment (larswik) [reply IFC4x3/Properties/l/LongitudinalSpacing_2Fqc$pMNj5C8jRJPnj8nSC/DocProperty.xml:2]: The domain experts wanted to keep it.

File: IFC4x3/Properties/l/LongitudinalConnections_1azFwlJa58zglMh9JjImfr/DocProperty.xml (hunk 3 above)
Comment (larswik) [reply IFC4x3/Properties/l/LongitudinalConnections_1azFwlJa58zglMh9JjImfr/DocProperty.xml:2]: Ping @Michel-Rives; Can you write a description?

**Comment by larswik** [comment]
@Sergej: I added missing descriptions in [67b2bf8](https://github.com/bSI-InfraRoom/IFC-Specification/pull/829/commits/67b2bf8db628cd663ad40d8ebfa407b3c55491f5) and [1ab2284](https://github.com/bSI-InfraRoom/IFC-Specification/pull/829/commits/1ab22843bac864bcda22ca66492500b70c58f4a3).

A couple of properties should be reused from #818.

**Comment by larswik** [comment]
Ping @SergejMuhic : Can hopefully be merged now.

**Review by SergejMuhic — APPROVED** [review]
For the record, I don't think that the bolt properties are at the right place.

**Pull request merged**